tests/CMakeLists.txt

```diff
@@ -263,6 +263,24 @@ if(WITH_PYTHON)
     add_fdb_test(TEST_FILES fast/MockDDReadWrite.toml)
   endif()
 
+  if (NO_MULTIREGION_TEST)
+    add_fdb_test(TEST_FILES fast/ValidateStorage.toml IGNORE)
```

Comment:
will this test be enabled if rocksdb is on?

Reply:
If there are multiple factors impacting the final state of a test, it might be good to combine all the factors in the same place to avoid surprise in the future.